schemas/constructs/openapi/relationshipEvaluation.yml

```diff
@@ -0,0 +1,48 @@
+openapi: 3.0.0
```

Comment:
Should we be using this be 3.1.0?

Reply:
@MUzairS15 to confirm whether oapi-codegen supports the 3.1.0 draft. By default, it doesn't.

Reply:
other schemas are also using openapi 3.0.0 right now